### PR TITLE
RHIDP-984: Fixed GitHub login failure

### PR DESCRIPTION
**Pull request by hmanwani-rh**
Jira: https://issues.redhat.com/browse/RHIDP-984

### CODE REVIEW

File: installation/proc-install-rhdh-helm.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ For more information about creating a project in OpenShift, see link: https://do
 . Click *Create*.
 . Copy the OpenShift router host (for example: `apps.<clusterName>.com`) to *Root Schema* -> *global* -> *Shorthand for users who do not want to specify a custom HOSTNAME. Used ONLY with the DEFAULT upstream.backstage.appConfig value and with OCP Route enabled.*, and adjust the other values if needed.
```
Comment (kadel) [inline installation/proc-install-rhdh-helm.adoc:30]: This also needs to be updated.

```suggestion
. Copy the OpenShift router host (for example: `apps.<clusterName>.com`) to *Root Schema* -> *global* -> *Custom hostname shorthand, overrides `global.clusterRouterBase`, `upstream.ingress.host`, `route.host`, and url values in `upstream.backstage.appConfig`*, and adjust the other values if needed.
```

Comment (hmanwani-rh) [reply installation/proc-install-rhdh-helm.adoc]: Done

**Comment by kadel** [comment]
I've just tried this. And it looks like it doesn't work as expected. 
Lets not merge this one yet.

**Comment by coreydaley** [comment]
From my explorations, the `global.clusterRouterBase` needs to be updated to reflect your base cluster domain, the `apps.<cluster>.com` url, unless you have configured a custom domain for your cluster, which is not the case when bringing one up via Cluster Bot (or by default probably). I think I have addressed this issue in https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/69 if you would like to take a look at it.

**Comment by hmanwani-rh** [comment]
> From my explorations, the `global.clusterRouterBase` needs to be updated to reflect your base cluster domain, the `apps.<cluster>.com` url, unless you have configured a custom domain for your cluster, which is not the case when bringing one up via Cluster Bot (or by default probably). I think I have addressed this issue in #69 if you would like to take a look at it.

@kadel PR #69 is merged, should we close this one then?

**Pull request closed without merging**